sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -446,16 +447,9 @@ SUBROUTINE CLDRAD
 !     TOTAL COLUMN AOD (TAOD553D FROM HRRR-SMOKE)
 !
       IF (IGET(735) > 0) THEN
-       IF (MODELNAME == 'RAPR') THEN
+       IF (MODELNAME == 'RAPR' .OR. MODELNAME == 'FV3R') THEN
          CALL CALPW(GRID1(ista:iend,jsta:jend),19)
          CALL BOUND(GRID1,D00,H99999)
-       ELSE IF (MODELNAME == 'FV3R') THEN
-         GRID1=SPVAL
-         DO J=JSTA,JEND
-           DO I=ISTA,IEND
-             if (AODTOT(I,J) < SPVAL) GRID1(I,J) = AODTOT(I,J)
```

Comment:
@EricJames-NOAA You add changes for calculating column aod from the 3D field in UPP. Do you think the capability of reading aodtot from model output is needed to be kept in UPP?

Reply:
@WenMeng-NOAA I'm OK with getting rid of this code; we can easily add it back in if needed.  Should I remove it?

Reply:
@EricJames-NOAA That would be great if you can remove aodtot related code. Thanks!

Reply:
@EricJames-NOAA Would you like the AODTOT clean up in this PR or next PR?

Reply:
@WenMeng-NOAA sorry for the delay.  I have the cleanup ready, I just need to commit the changes.  Stand by...